src/client/__init__.py

```diff
@@ -1389,49 +1396,49 @@ def clone_pool(self, pool_id: str) -> Pool:
 
     @add_headers('client')
     def clone_pool_async(self, pool_id: str) -> operations.PoolCloneOperation:
-        """Duplicates existing pool, asynchronous version
+        """Clones an existing pool. Sends an asynchronous request to Toloka.
 
-        An empty pool with the same parameters will be created.
-        A new pool will be attached to the same project.
+        An empty pool with the same parameters is created.
+        The new pool is attached to the same project.
 
         Args:
-            pool_id: ID of the existing pool.
+            pool_id: The ID of the pool to be cloned.
 
         Returns:
-            PoolCloneOperation: An operation upon completion of which you can get the new pool.
+            PoolCloneOperation: An object to track the progress of the operation.
 
         Example:
-            >>> new_pool = toloka_client.clone_pool_async(pool_id='1')
-            >>> toloka_client.wait_operation(new_pool)
+            >>> clone_op = toloka_client.clone_pool_async(pool_id='1')
+            >>> toloka_client.wait_operation(clone_op)
             ...
         """
         response = self._request('post', f'/v1/pools/{pool_id}/clone')
         return structure(response, operations.PoolCloneOperation)
 
     @add_headers('client')
     def create_pool(self, pool: Pool) -> Pool:
-        """Creates a new pool
+        """Creates a new pool in Toloka.
 
         You can send a maximum of 20 requests of this kind per minute and 100 requests per day.
 
         Args:
-            pool: New Pool with set parameters.
+            pool: The pool to be created.
 
         Returns:
-            Pool: Created pool. With read-only fields.
+            Pool: The pool with updated read only parameters.
```

Comment:
`read-only` should be hyphenated (see https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/read-only)

Reply:
Fixed _read-only_ in my PR.

There are examples without a hyphen in other files. I did not touch them.